pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -57,6 +58,7 @@ import (
 
 const (
 	controllerName = "mxnet-operator"
+	frameworkName  = "mxnet"
```

Comment:
Can we move this const here

https://github.com/kubeflow/tf-operator/blob/2712f5667ec78f17d22288630f8719f0c08990ba/pkg/apis/mxnet/v1/constants.go#L19-L21 ?

Reply:
same for rest of frameworks

Reply:
@deepak-muley

Reply:
Done.